src/puppetlabs/trapperkeeper/services/metrics/metrics_core.clj

```diff
@@ -19,13 +21,21 @@
 (def JmxReporterConfig
   {:enabled schema/Bool})
 
+(def JolokiaApiConfig
+  {(schema/optional-key :enabled) schema/Bool
+   (schema/optional-key :config) jolokia/JolokiaConfig})
```

Comment:
Oh derp. Yeah, it should be. Guess there should also be a test that would have tripped that :)